docs/manifest.md

```diff
@@ -164,6 +164,9 @@ Internally we validate each submitted manifest using the following schema
 [Manifest JSON schema v2.0](schema/2.0/manifest.schema.json)
 
 A manifest can be validated using the [online tool](https://www.jsonschemavalidator.net/)
+### identifier
+
+Unique identifier of the application as defined by the application creator. The identifier may contain uppercase or lowercase letters ('A' through 'Z'), numbers, underscores ('_'), hyphens('-') and dots('.'). The minimum length is 6 characters a the maximum is 128.
```

Comment:
Btw, we currently allow [only hyphens in UI](https://github.com/getoutreach/client/blob/master/packages/composites/modals/src/components/MarketplaceCreateApplicationModal/components/CreateAppForm.tsx#L17-L20).

Reply:
I suggest you allow these additional characters in UI as well!